tools/changelog

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
```

Comment:
is this script taken from somewhere? Also should the file be named  changelog.sh

Reply:
No, just something handwritten for our use case. As a general rule, I prefer to omit the extension on scripts which are marked executable and meant to be used as utility programs, as this and `package` are; and add the extension when they are meant to be sourced/included in other scripts (i.e. like libraries). It's a subjective rule, but I find it works well since it doesn't matter to the consumer of the script what it's written in, and virtually every editor I know of understands how to syntax highlight shell scripts with the shebang, so there is little objective reason to require it. That said, it doesn't matter too much to me, so if we prefer a different approach, just let me know

Reply:
ok that works.

Reply:
ok that works